### ISSUE TITLE
periodic boundaries

### ISSUE THREAD

**Issue by suessmann** [agayonlombardo/threephase_porous_material#1]
Hello! 

Thanx for the paper, it is full of amazing results. One of them is the way you managed to generate periodic boundaries! However, I could not find any trace of it in the code available. 

In the paper it is stated that we need to apply circular padding, but you did not go into the details. What size should it be? Is it Transposed padding or just usual padding from `torch.nn.functional.pad`? I tried to replicate you code and succeeded in non-periodic part, but I still cannot manage to generate clean periodic boundaries. May I ask you to clarify on this, please?

Ilya

**Comment by suessmann** [comment]
It took some time to think about the problem and now I got how you achieved periodic boundaries :)
